radar-gateway/src/main/kotlin/org/radarbase/gateway/utils/Env.kt

```diff
@@ -0,0 +1,10 @@
+package org.radarbase.gateway.utils
+
+object Env {
+    const val S3_SERVICE_URL = "S3_SERVICE_URL"
+    const val S3_ACCESS_KEY = "S3_ACCESS_KEY"
```

Comment:
Can we use the same names as recommended by AWS for CLI - https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-envvars.html ?